src/client/containers/BoardPageComponents/BoardAddCandidate/BoardHeader.style.css

```diff
@@ -0,0 +1,37 @@
+* {
+  box-sizing: border-box;
+}
+
+.Board-container {
```

Comment:
We have many board components by now. You have to specify what the css classes deal with.
board-header-container

Same below

Reply:
okk